addons/api/fnc_isRadio.sqf

```diff
@@ -16,14 +16,10 @@
  */
 
 params [
-    ["_weapon", "", [""]]
+    ["_radio", "", [""]]
 ];
 
-// check that it at least inherits from ACRE_BaseRadio *AND* has a unique ID
-// If its scoped 1, that means its either a base or not a radio.
-private _flag = getNumber (configFile >> "CfgWeapons" >> _weapon >> "acre_isRadio");
-private _id = getNumber (configFile >> "CfgWeapons" >> _weapon >> "acre_uniqueId");
```

Comment:
Where did that go? I'd also keep the comment above.

Reply:
the comment above looks like from a version where the code worked differently because
1. `getNumber` never returns nil so the `if isNil exitWith` would never trigger
2. the comment says that if it is scoped 1 it is a base radio or not a radio what is not reflected in the code below because there  it actually got check if it was one

so that comment basically contradicts what the code did before and also the Function Description says.

the code now returns true on Base Class Radios for example an ACRE_PRC117F but also on an ACRE_PRC117F_id_1.

before it only returned true on an ACRE_PRC117F_id_1 but not on an ACRE_PRC117F

Reply:
Thinking about this I think this function should only return `true` for actual unique ID'd radios. Only those are actual "radios", base classes are just unusable items in ACRE's world. So I would modify the header to reflect that and keep the logic the same for that part.

I guess scope part is redundant.